compiler/rustc_infer/src/infer/error_reporting/nice_region_error/trait_impl_difference.rs

```diff
@@ -23,51 +23,45 @@ impl<'a, 'tcx> NiceRegionError<'a, 'tcx> {
         let error = self.error.as_ref()?;
         debug!("try_report_impl_not_conforming_to_trait {:?}", error);
         if let RegionResolutionError::SubSupConflict(
-            _,
-            var_origin,
-            sub_origin,
-            _sub,
-            sup_origin,
-            _sup,
-            _,
-        ) = error.clone()
+                _,
+                var_origin,
+                sub_origin,
+                _sub,
+                sup_origin,
+                _sup,
+                _,
+            ) = error.clone()
```

Comment:
Is this indentation too far in? I wouldn't have expected this to change.

Reply:
Folded these in a single line